apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -437,7 +438,10 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     element = lxml.html.fragment_fromstring(element_html)
     answer_name = pl.get_string_attrib(element, "answers-name")
     format = pl.get_enum_attrib(element, "format", FormatType, FormatType.DEFAULT)
-
+    inline = pl.get_boolean_attrib(element, "inline", INLINE_DEFAULT)
+    dropzone_layout = pl.get_string_attrib(
```

Comment:
It seems like this variable is only supposed to take on a finite set of values? In that case, should use `get_enum_attrib` instead of `get_string_attrib` with an appropriately defined enum.

Also, be sure to update the docs.

Reply:
This attribute predates this PR so I didn't want to mess with it initially. @SethPoulsen is there any reason it was a string attribute or should I go ahead and switch to an `enum`?

Reply:
dropzone layout can definitely be an enum, it just hasn't been converted yet. (The enum system is new).

Reply:
Cool, updated. :)